exercises/k9-data-structures/lists/linked-list-vs-array/shared/test-suite.js

```diff
@@ -7,7 +7,8 @@ export function ListTestSuite(factory) {
             expect(list.get(0)).toBe(0);
             expect(list.get(1)).toBe(1);
             expect(list.get(2)).toBe(2);
-            expect(list.get(3)).toBe(null);
+            expect(() => { list.get(3) }).toThrow();
```

Comment:
You need to run the array tests again, I'm not sure they throw 😉 

If we're changing the contract to throw, we should make assertions about what that error is. Maybe we should have throw a specific error when accessing an undefined index: `expect(() => { list.get(3) }).toThrow("index out of bounds")`. That said throwing on accessing a value can be dangerous if there is no way to check it exists–if you wanted to take this approach further try implementing a `indexExists` function allow consumers to check before accessing.

Reply:
Can you run through this with me quickly at some point? I thought I had checked the test, clearly not 🙈